samples/SocketsSample/Hubs/TestHub.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+
+namespace SocketsSample.Hubs
+{
+    public class TestHub : Hub<IChatClient>
+    {
+        public override Task OnConnectedAsync()
+        {
+            Clients.All.JoinChat();
+            return Task.CompletedTask;
```

Comment:
I realize this is WIP, but you're definitely going to need a test that does real async (using Task.Yield or Task.Delay)